packages/verification/src/models/v1/mod-callbacks/callout-request-event-response/callout-request-event-response.ts

```diff
@@ -0,0 +1,23 @@
+import { ActionEnum } from '../../enums';
+
+export interface CalloutRequestEventResponse {
```

Comment:
Not related to this PR but 'callout*' OAS schemas was renamed to 'PhoneCall*' and documentation is referencing to `Phone Call` for `Callout`
(here new name is `PhoneCallRequestEventResponse`)